scripts/purge-s3.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -euox pipefail
+
+echo 'Warning: this action will completely remove all the documents in'
+echo 'both the trusted and untrusted S3 buckets of '${KUBE_NAMESPACE^^}'. It cannot be undone!'
```

Comment:
You aren't gating the namespace in these latter scripts. Did you mean to?

Reply:
This is intentional as the ES & scale scripts run externally for now, while the others use the KUBE_NAMESPACE variable.